login/token_context/token.go

```diff
@@ -0,0 +1,32 @@
+// Package token_context contains the code that extract token manager from the
+// context.
+package token_context
```

Comment:
Hm, I thought @aslakknutsen asked why not using the existing token package instead of introducing login/token_context? Not about renaming the name login/token_context package to a new login/token package. Because having login/token and token packages is confusing.